packages/cli-doctor/src/tools/healthchecks/xcodeEnv.ts

```diff
@@ -0,0 +1,61 @@
+import {HealthCheckInterface} from '../../types';
+import fs from 'fs';
+import path from 'path';
+import {promisify} from 'util';
+import {findProjectRoot} from '@react-native-community/cli-tools';
+import findAllPodfilePaths from '@react-native-community/cli-platform-ios/src/config/findAllPodfilePaths';
```

Comment:
We can't import from `src` since we transform the code before publishing and put it into `build` directory; `src` is not there in published files. We either need to export `findAllPodfilePaths` from `@react-native-community/cli-platform-ios` – and add a dependency on it for `cli-doctor` and update `"references"` in its `tsconfig.json` – or move this helper to the `cli-tools` package (and add a dependency on `glob`). I'd go with the former for now.

Reply:
FYI we have a `yarn watch` script that builds files on the go

Reply:
Hi @thymikee, thanks for the review.
I updated the PR with your suggestion. Let me know if it's ok!